library/src/test/java/io/github/luizgrp/sectionedrecyclerviewadapter/testdoubles/stub/ParametersedSectionStub.java

```diff
@@ -0,0 +1,32 @@
+package io.github.luizgrp.sectionedrecyclerviewadapter.testdoubles.stub;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import io.github.luizgrp.sectionedrecyclerviewadapter.Section;
+import io.github.luizgrp.sectionedrecyclerviewadapter.SectionParameters;
+
+/**
+ * A stub of Section constructed using SectionParameters
+ */
+public class ParametersedSectionStub extends Section {
```

Comment:
I see that you had to create this class in order to unit test the new constructor but unfortunately this is not really a stub. 
Please remove this class and create the following method at the bottom of `SectionTest` to be used with your `build_constructsCorrectSection` test:

    private Section getSection(SectionParameters sectionParameters) {
        return new Section(sectionParameters) {
            @Override
            public int getContentItemsTotal() {
                return 0;
            }

            @Override
            public RecyclerView.ViewHolder getItemViewHolder(View view) {
                return null;
            }

            @Override
            public void onBindItemViewHolder(RecyclerView.ViewHolder holder, int position) {

            }
        };
    }